pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -186,15 +187,15 @@ func NewVolumeBinder(
 	storageClassInformer storageinformers.StorageClassInformer,
 	bindTimeout time.Duration) SchedulerVolumeBinder {
 	return &volumeBinder{
-		kubeClient:      kubeClient,
-		podLister:       podInformer.Lister(),
-		classLister:     storageClassInformer.Lister(),
-		nodeInformer:    nodeInformer,
-		csiNodeInformer: csiNodeInformer,
-		pvcCache:        NewPVCAssumeCache(pvcInformer.Informer()),
-		pvCache:         NewPVAssumeCache(pvInformer.Informer()),
-		bindTimeout:     bindTimeout,
-		translator:      csitrans.New(),
+		kubeClient:    kubeClient,
+		podLister:     podInformer.Lister(),
+		classLister:   storageClassInformer.Lister(),
+		nodeLister:    nodeInformer.Lister(),
+		csiNodeLister: csiNodeInformer.Lister(),
```

Comment:
we should hold the lister b/c the shared informer will not be initialized if the `Lister()/Informer()` is not called,
otherwise, the informer will not be started by `SharedInformerFactory.Start`.

though these informers normally will be initialized in other components, this can fix the potential bug